packages/@react-types/shared/src/collections.d.ts

```diff
@@ -189,20 +189,22 @@ export interface Node<T> {
   /** A unique key for the node. */
   key: Key,
   /** The object value the node was created from. */
-  value: T | null,
+  value: T | null | undefined,
   /** The level of depth this node is at in the hierarchy. */
   level: number,
   /** Whether this item has children, even if not loaded yet. */
-  hasChildNodes: boolean,
+  hasChildNodes?: boolean,
```

Comment:
guess this has always been technically optional due to partialNode's type?

Reply:
it's been 4+ months, i have no recollection of why I decided to do this here as opposed to like the previous one and consider it to be an invalid element